apps/bfd-model/bfd-model-rif-samples/src/test/java/gov/cms/bfd/model/rif/samples/SampleDataColumnsTest.java

```diff
@@ -103,8 +102,8 @@ private void verifyColumns(StaticRifResourceGroup sampleGroup) {
                   toHeaderFormat(columnsInEnum, c -> c.name())));
         }
       }
-    } catch (IOException e) {
-      throw new UncheckedIOException(e);
+    } catch (UncheckedIOException e) {
+      throw e;
```

Comment:
Not exactly sure what the point of this statement is anyway; if it didn't exist it'd do the same thing.

Reply:
This is why I made this change. One of the dependencies I upgraded caused it to fail. <img width="1469" alt="Screenshot 2023-02-20 at 2 09 37 PM" src="https://user-images.githubusercontent.com/10262289/220205608-14a0b68d-3152-4674-a960-0a6b4969faa6.png">

Reply:
I'm suggesting just take that catch out completely, it did (and continues to do) nothing.

Reply:
Removed